linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/ansible/shared.yml

```diff
@@ -11,8 +11,13 @@
   register: user_names
 
 - name: Change the maximum time period between password changes
+{{% if product not in ["rhel7", "ol7"] %}}
   ansible.builtin.user:
     user: '{{ item }}'
     password_expire_max: '{{ var_accounts_maximum_age_login_defs }}'
+{{% else %}}
+  ansible.builtin.command: >
+    chage -M 60 {{ item }}
```

Comment:
Should 60 be replaced with `var_accounts_maximum_age_login_defs`?

Reply:
It should. I will fix. Thanks.